proposals/0167-app-services.md

```diff
@@ -433,10 +434,58 @@ The `AppServiceManifest` is essentially detailing everything about a particular
 
 	</struct>
 	
+	
+	<struct name="ServiceAction">
+	    <param name="baseUri" type="String" mandatory="true">
+	        <description> Also referred to as the authority and path of the URI. The base URI of this action. If no params are required, this plus the URI prefix can be used as the entire URI (uriPrefix://baseUri?paramKey=paramValue&paramKey=paramValue#paramKey=paramValue). If this param is left as an empty set, params will be assumed to be part of the path of the URI with no keys and the fully qualified URI will be constructed accordingly (uriPrefix:param:param:param).</description>
+	    </param>
+	    
+	    <param name="queryParameters" type="Parameter" array="true" mandatory="false">
+	        <description>An array of parameters for this action that should be included as part of the query string of the URI. When sent as a fully qualified URI these parameters will be attached in accordance to RFC3986 https://tools.ietf.org/html/rfc3986</description>
+	    </param>
+	    
+	    <param name="fragmentParameters" type="Parameter" array="true" mandatory="false">
+	        <description>An array of parameters for this action that should be included in the fragment section of the URI. When sent as a fully qualified URI these parameters will be attached in accordance to RFC3986 https://tools.ietf.org/html/rfc3986</description>
+	    </param>
+	    
+	    <param name="isQuickAction" type="Boolean" mandatory="false">
+	        <description>If set to true, this implies this action can be acted upon without an additional parameters or actions. Most common use case would be a consumer displaying a soft button to represent this action</description>
```

Comment:
Update to:	     
   <description>If set to true, this implies this action can be acted upon **without additional** parameters or actions. **The** most common use case would be a consumer displaying a soft button to represent this action</description>